packages/tabs/src/TabsOverflow.ts

```diff
@@ -51,6 +51,12 @@ export class TabsOverflow extends SizedMixin(SpectrumElement) {
     @property({ reflect: true })
     public override dir!: 'ltr' | 'rtl';
 
+    @property({ reflect: true, type: String, attribute: 'label-previous' })
+    public labelPrevious: string | undefined;
```

Comment:
if this can be defined can this be:
```suggestion
    public labelPrevious?: string;
```

Reply:
If the next and previous buttons should not be keyboard accessible, is it necessary to introduce these labels? Instead, should we ensure that by keyboard, the next and previous buttons are not interacted with?

Reply:
I think that screenreader labeling can still be useful even without keyboard accessibility. (thinking of mobile users, who use tap controls and screenreaders read out what is tapped)